pom.xml

```diff
@@ -54,16 +54,10 @@
             <artifactId>workflow-step-api</artifactId>
             <version>2.12</version>
         </dependency>
-        <dependency>
-            <groupId>org.jenkins-ci.plugins</groupId>
-            <artifactId>variant</artifactId>
-            <version>1.1</version>
-        </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
             <version>2.22</version>
-            <optional>true</optional>
```

Comment:
@jglick why did you do this without removiong the exclusion on scm-api.

This breaks any consumer plugin of junit plugin and forces them to declare (an uneeded) dependency on scm-api so that hpi:run and the basic InjectedTest start the plugin works.